ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -0,0 +1,96 @@
+'''
+Tests for ph5tostationxml
+'''
+import unittest
+import os
+import sys
+from mock import patch
+
+from testfixtures import OutputCapture
+
+from ph5.clients import ph5tostationxml
+from ph5.core.tests.test_base import LogTestCase, TempDirTestCase, \
+     kef_to_ph5
+
+
+class TestPH5toStationXMLParser_main(LogTestCase, TempDirTestCase):
+    def test_main(self):
+        # array_multideploy: same station different deploy times
+        # => check if network time cover all or only the first 1
+        kef_to_ph5(self.tmpdir,
+                   'master.ph5',
+                   os.path.join(self.home, "ph5/test_data/metadata"),
+                   ["array_multi_deploy.kef", "experiment.kef"])
+        testargs = ['ph5tostationxml', '-n', 'master',
+                    '--level', 'network', '-f', 'text']
+        with patch.object(sys, 'argv', testargs):
+            with OutputCapture() as out:
+                ph5tostationxml.main()
+                output = out.captured.strip().split("\n")
+        self.assertEqual(
+            output[1],
+            "AA|PH5 TEST SET|2019-06-29T18:08:33|2019-09-28T14:29:39|1")
```

Comment:
Use context variable `out` inside the context manager

```suggestion
        with patch.object(sys, 'argv', testargs):
            with OutputCapture() as out:
                ph5tostationxml.main()
                output = out.captured.strip().split("\n")
                self.assertEqual(
                    output[1],
                    "AA|PH5 TEST SET|2019-06-29T18:08:33|2019-09-28T14:29:39|1")
```